### PR TITLE
Trying to fix output dir option

### PR DESCRIPTION
**Pull request by liuzhenqi77**
Hey I'm trying to use the `output_dir` option, which is not working as expected right now. 

This is a draft fix following tractoflow. Please feel free to edit. Thanks!

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -205,7 +205,7 @@ bundle_for_average
     .groupTuple(by: 0)
     .set{all_bundle_for_average}
 process Average_Bundles {
-    publishDir = "./results_rbx/Average_Bundles/"
```
Comment (GuillaumeTh) [inline main.nf:208]: @frheault any reason to publish Average_Bundles in results_rbx instead of output_dir when necessary ?

Comment (frheault) [reply main.nf:208]: No reason, my mistake.

File: main.nf [hunk 2]
```diff
@@ -205,7 +205,7 @@ bundle_for_average
...
+    publishDir = params.Average_Bundles_Publish_Dir
```
Comment (frheault) [inline main.nf:208]: I think that we have an unwritten convention of params always being in lower case?
@GuillaumeTh

**Review by frheault — COMMENTED** [review]
Small comment, thx for fixing this !

File: main.nf (hunk 2 above)
Comment (GuillaumeTh) [reply main.nf:208]: No that's ok. We do the same thig in TractoFlow

**Review by GuillaumeTh — APPROVED** [review]
Looks good to me. I test it and merge after.

**Pull request merged**